INSTALL.md

```diff
@@ -70,6 +70,10 @@ settings that normally require modification.
 Start the server with `sudo systemctl start owntone` and check that it is
 running with `sudo systemctl status owntone`.
 
+You can start or enable the service for a "zone" by `sudo systemctl
```

Comment:
This is specialised use (+ difficult to understand what a zone is), so I still would like it removed from the "Quick version". Same below. Other than that it looks good.

Reply:
Okay, I migrated everything to README.md (and updated the README a bit).

If you're okay with the results I can re-squash (unless you're willing to do that during the merge).